sys/Makefile.dep

```diff
@@ -6,8 +6,7 @@ ifneq (,$(filter arduino,$(USEMODULE)))
   FEATURES_REQUIRED += cpp
   FEATURES_REQUIRED += periph_gpio
   FEATURES_REQUIRED += periph_uart
-  SKETCH_MODULE ?= arduino_sketches
-  USEMODULE += $(SKETCH_MODULE)
+  USEMODULE += arduino_sketches
```

Comment:
It seems like this does more than 'fix kconfig of arduino', probably worth its own commit at least.